packages/core/src/lib/options.types.ts

```diff
@@ -1,7 +1,7 @@
 export type NetworkId = "mainnet" | "testnet";
 
 export interface Network {
-  networkId: string;
+  networkId: NetworkId;
```

Comment:
Why are we changing the type here from `string` to `NetworkId`?

This will limit the network id for custom networks  to just either "testnet" or "mainnet"

```ts
const selector = await setupWalletSelector({
    network: {
      networkId: "localnet",
      nodeUrl: "http://127.0.0.1:52993",
      helperUrl: "http://127.0.0.1:52997",
      explorerUrl: "http://127.0.0.1:53009",
      indexerUrl: "http://127.0.0.1:52997",
    },
    modules: [setupNearWallet()],
  });
```

Typescript would throw an error: **_TS2322: Type '"localnet"' is not assignable to type 'NetworkId'_**

Reply:
Initially my thoughts were, leaving `networkId` as `string` isn't really helpful to anyone. And then saw `NetworkId` type above so I assume that this was left as backlog. (My mistake)

If we are officially supporting `localnet` and any other network, should be update `NetworkId` type and use it instead of string? (eg. `export type NetworkId = "mainnet" | "testnet" | "localnet";`)

Reply:
The idea is that we have presents only for testnet and mainnet  so NetworkId is used for that reason, if a developer wishes to use another network they can provide an object of type Network.

Reply:
Thanks for the correction, reverted to where it was